image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [654.64588,   0.     , 316.31643,
           0.     , 871.91715, 242.22718,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.160487, -0.287598, 0.003915, 0.000131, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [669.54327,   0.     , 315.86919,   0.     ,
           0.     , 889.5127 , 243.24275,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
